### ISSUE TITLE
Cannot install npm

### ISSUE THREAD

**Issue by TheGoldenEye** [emostov/staking-payouts#36]
Unfortunately the latest npm (v1.0.0) is not installable
```
xxx:~/staking-payouts$ sudo npm install -g @zekemostov/staking-payouts
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /usr/local/lib/node_modules/@zekemostov/staking-payouts/build/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/@zekemostov/staking-payouts/build/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-05-04T11_45_34_920Z-debug.log

```

Installing with `yarn` also fails, but without error message

**Comment by emostov** [comment]
Thanks for the bug report. I just made another release that should fix the distribution bundle. https://github.com/emostov/staking-payouts/releases/tag/v1.0.1

**Comment by emostov** [comment]
Closing because this issue should have been fixed with release v1.0.1.
